platform/Jwt.roc

```diff
@@ -0,0 +1,89 @@
+module [
+    Err,
+    Algorithm,
+    verify,
+]
+
+import PlatformTasks
+
+Err : [
+    AlgorithmMismatch,
+    InvalidSignature,
+    BadFormat,
+    BadJson,
+    MissingClaims,
+    MissingHeader,
+    MissingKeyId,
+    MissingSignature,
+    Other Str,
+]
+
+Algorithm : [
+    Hs256,
+    Hs384,
+    Hs512,
+    # TODO add support for other algorithms
+    # See https://docs.rs/jwt/0.16.0/jwt/index.html for more information
+    # Note: at this time, by default only the `hmac` crate’s Hmac type is supported.
+    # Rs256,
+    # Rs384,
+    # Rs512,
+    # Es256,
+    # Es384,
+    # Es512,
+    # Ps256,
+    # Ps384,
+    # Ps512,
+    # Unsecured,
+]
+
+algoToU8 : Algorithm -> U8
```

Comment:
Not needed for this PR, but I would really love to clean up here. Neither `algoToU8` or `mapErrFromHost` from host should exist. We have working glue generation for tags. We should use it.

Reply:
Done, removed and using proper tag union.